ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -996,6 +1025,10 @@ + (BOOL)supportsSecureCoding {
     return YES;
 }
 
+- (NSString*)stringForAnswer:(id)answer {
+    return ORKTimeOfDayStringFromComponents(answer);
+}
```

Comment:
@umerkhan-apple I've just realized that `ORKTimeOfDayStringFromComponents` uses a plain `NSDateComponentsFormatter`. That should allow for proper localization out of the box. I tried in the simulator (en_US) and on a device (de_DE), and that worked. So why should this be localization unfriendly?